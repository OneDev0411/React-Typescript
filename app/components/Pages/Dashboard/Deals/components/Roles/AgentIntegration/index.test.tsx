import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import { RoleAgentIntegration } from '.'

describe('Deal Agent integration with roles', () => {
  it('renders the component', () => {
    const wrapper = shallow(
      <RoleAgentIntegration
        isPrimaryAgent
        onUpsertRole={() => null}
        onClose={() => null}
      />
    )

    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
