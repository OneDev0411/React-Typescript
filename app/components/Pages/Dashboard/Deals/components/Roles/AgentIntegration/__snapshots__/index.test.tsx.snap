// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deal Agent integration with roles renders the component 1`] = `
<Fragment>
  <DealRoleModal
    form={null}
    isOpen={true}
    onClose={[Function]}
    onUpsertRole={[Function]}
  />
</Fragment>
`;
